virtool/http/headers.py

```diff
@@ -12,3 +14,16 @@ async def headers_middleware(req, handler):
     resp.headers["Server"] = "Virtool"
 
     return resp
+
+
+async def format_location(request: Request, response: Response):
```

Comment:
Change to `on_prepare_location` so we known this is a signal handler elsewhere in code.